packages/amesos2/src/Amesos2_MatrixAdapter_def.hpp

```diff
@@ -249,14 +265,50 @@ namespace Amesos2 {
 				     EStorage_Ordering ordering,
 				     no_special_impl nsi) const
   {
-  
+
     //Added void to remove parameter not used warning
     ((void)nsi);
```

Comment:
The proper idiom is not to name the parameter:
```c++
void my_function(int used_parameter, double /* unused_parameter */);
```
Some compilers will warn even if you cast to `void`, so the best thing to do is not to name the parameter in the function's definition.  If you have a separate declaration, you may name the parameter in the declaration.